app-text/master-pdf-editor/master-pdf-editor-4.3.61.ebuild

```diff
@@ -0,0 +1,73 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit xdg-utils eutils versionator
+
+DESCRIPTION="Master PDF Editor is the complete solution for viewing and editing PDF files."
+HOMEPAGE="https://code-industry.net/free-pdf-editor/"
+
+SRC_URI="http://get.code-industry.net/public/${P}_qt5.amd64.tar.gz"
+
+RESTRICT="mirror"
```

Comment:
Small nitpick: The usual convention is for RESTRICT to go to the end of the second variables block (i.e., LICENSE, SLOT, KEYWORDS, [IUSE], [RESTRICT]).

Reply:
Will do, thanks!